common/sign/v1/tests/test_iss_kerl.c

```diff
@@ -77,6 +79,43 @@ void test_key(trit_t *s, trit_t *k, size_t l, Kerl *c) {
   "CFOYUCXLHLSUBAEYOTAWUNRPJFA9TSJNLBFLMZQASPTVCMTFBOQQRGGQ9MRZCJWYGBORJZQWVS" \
   "BLVKBVW"
 
+#define EXP_SIG                                                                \
+  "DQMAVYJTIIQUCDHRIZAPYKFCFBTHNDAMHLYEEIEBSR9REQBBVUQOSDWFSGPTZFNQQTVMQPVUDH" \
+  "DTEZVSW"
+
+void test_sig_digest() {
+  const size_t key_length = ISS_KEY_LENGTH;
+  trit_t seed[HASH_LENGTH_TRIT];
+  trit_t subseed[HASH_LENGTH_TRIT];
+  trit_t key[key_length];
+  trit_t *k = key;
+  const size_t l = ISS_KEY_LENGTH;
+
+  Kerl kerl;
+  Kerl* c = &kerl;
+  init_kerl(&kerl);
+
+  tryte_t tryte_seed[] = TEST_SEED2;
+  trytes_to_trits(tryte_seed, seed, 81);
+  iss_kerl_subseed(seed, subseed, 0, c);
+  iss_kerl_key(subseed, k, l, c);
+
+  tryte_t addy_trytes[l];
+  trit_t s[6561];
+  trit_t d[6561];
+  memcpy(s, k, sizeof(trit_t)*6561);
+
+  iss_kerl_sig_digest(d, s, k, l, c);
```

Comment:
You're using the start of the key as hash  for the digest ?

Reply:
yes